index.dd

```diff
@@ -94,6 +94,25 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE Start a minimal web server,
+----
+#!/usr/bin/env dub
+/+ dub.sdl:
+name "hello_vibed"
```

Comment:
FYI: With newer DUB instances this line won't be needed anymore (-> https://github.com/dlang/dub/pull/1081), but for now we have to wait a couple of months until https://github.com/dlang/dub/pull/1081 has gone through a couple of releases.